### PR TITLE
Rudy feature link props

### PR DESCRIPTION
**Pull request by rudyesar**
- Fixed carousel image aspect-ratio
- added functionality so item page compatible with path params

### CODE REVIEW

File: client/src/itemView/CarouselContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+const CarouselContainer = (props) => {
+
+  const carouselItem = props.images.map((image, i) =>
+    <Carousel.Item key={i} style={{height: '30em'}}>
+      <img
+        className='d-block w-100'
+        // className='img-fluid'
+        src={image}
+        style={{height: '30em', width: '100%', objectFit: 'contain'}}
```
Comment (joeyohie) [inline client/src/itemView/CarouselContainer.jsx:24]: is objectFit: 'contain' how you got the images to stay within the container?

**Review by joemitz — APPROVED** [review]
Looks good!

**Review by joeyohie — APPROVED** [review]
Way to go, Rudy! Lots of great work here and happy to see the router working with the item id :)

File: client/src/itemView/CarouselContainer.jsx (hunk 1 above)
Comment (rudyesar) [reply client/src/itemView/CarouselContainer.jsx:24]: Yup! so that the image maintains its aspect ratio

**Pull request merged**